VideoGenerator/example/app/src/test/java/de/guiframediff/videogeneratorexample/VideoGeneratorFileSystemTest.kt

```diff
@@ -0,0 +1,20 @@
+package de.guiframediff.videogeneratorexample
```

Comment:
Should we move this test to VideoGenerator/library, as it has nothing to do with the android app?